media/js/base/mozilla-fxa-iframe.js

```diff
@@ -55,6 +62,15 @@ Mozilla.FxaIframe = (function() {
         // initialize GA event name
         _gaEventName = _config.gaEventName || 'fxa';
 
+        if (_config.distribution && _config.distribution !== 'default') {
+            _distHost = $('#fxa-iframe-config').data(_config.distribution.toLowerCase() + '-host');
```

Comment:
Nit: the correct way to query data attributes is to use camelCase, so here it would be `.data(mozillaonlineHost)`. That said, this could get a little clunky with mixed casing depending on the `distribution` value. Would it perhaps be cleaner to drop the `-host` part?